pkg/sentry/platform/kvm/machine.go

```diff
@@ -776,32 +774,6 @@ func seccompMmapRules(m *machine) {
 		if err := sighandling.ReplaceSignalHandler(unix.SIGSYS, addrOfSigsysHandler(), &savedSigsysHandler); err != nil {
 			panic(fmt.Sprintf("Unable to set handler for signal %d: %v", bluepillSignal, err))
 		}
-		rules := []seccomp.RuleSet{
```

Comment:
It is the same as mentioned above. We currently revert KVM mmap trap mechanism as a work-around since we met a panic issue after importing cgo, which is explained in detail in https://github.com/google/gvisor/issues/9266#issuecomment-1855737336

Reply:
Andrei' patch: https://github.com/google/gvisor/commit/c2ab4cb4d9daf501c09ae3ac3a624a78825d8c8d does solve this issue. Thanks a lot.